tests/test/baseledger.test.js

```diff
@@ -0,0 +1,131 @@
+const request = require('supertest');
+const Web3 = require('web3')
+const fs = require('fs');
+const path = require("path");
+const chai = require("chai");
+const expect = chai.expect;
+
+const node1_api_url = 'localhost:1317';
+const node2_api_url = 'localhost:1318';
+const node3_api_url = 'localhost:1317';
+
+const baseledger_abi = JSON.parse(fs.readFileSync(path.join(__dirname, 'baseledger_abi.json')));
+
+const sleep = (ms) => {
+  return new Promise(resolve => setTimeout(resolve, ms));
+};
+const TEST_TIMEOUT = 30000;
+
+describe('validator power update', () => {
+  it('should add/update validator staking power', async function() {
+    this.timeout(TEST_TIMEOUT + 60000);
+    const orchestratorValidatorResponse = await request(node1_api_url).get('/Baseledger/baseledger/bridge/orchestrator_validator_address')
+        .send().expect(200);
+
+    // using first node validator to change staking
+    const parsedOrchValResponse = JSON.parse(orchestratorValidatorResponse.text);
+    const validatorAddress = parsedOrchValResponse.orchestratorValidatorAddress[0].validatorAddress;
+
+    let validators = await request(node1_api_url).get(`/cosmos/staking/v1beta1/validators/${validatorAddress}`)
+    .send().expect(200);
+
+    let parsedResponse = JSON.parse(validators.text);
+
+    console.log('validator tokens at start ', parsedResponse.validator.tokens);
+            
+    // add payee with 50k ubt
```

Comment:
this is the reason to clean containers between tests, to not add same payee in consecutive tests execution, because addPayee fails when there is payee already

maybe i can add call here to check for existing payee first and then call add/update depending on that - gonna try that out